geode-junit/src/main/java/org/apache/geode/codeAnalysis/AnalyzeSerializablesJUnitTestBase.java

```diff
@@ -88,7 +89,7 @@
   private File expectedDataSerializablesFile;
   private String expectedSerializablesFileName =
       "sanctioned-" + getModuleName() + "-serializables.txt";
-  private File expectedSerializablesFile;
+  private InputStream expectedSerializablesInputStream;
```

Comment:
Why is this InputStream a field on the class? Doesn't seem like something we should keep around or reuse.

Reply:
I've deleted this instance variable